src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
+  empower people with open data about their environments.
```

Comment:
Maybe we should list a few examples on what data is added?
I found only radiation and air quality, but judging from the api, it could be anything?

I think, it's okay if the descriptions are 2-3 lines long, maybe even a bit longer.

Reply:
I checked, but it's really only radiation (Cpm) and air quality (μSv/h) at the moment.
I'll add a sentence about it.